test/testlib.sh

```diff
@@ -594,3 +594,13 @@ cleanup_moreutils_parallel() {
     unlink "$ROOTDIR/test/bin/parallel"
   fi
 }
+
+# setup_actions_enabled_in_settings_json writes settings for the Actions app to settings.json
+# it accepts true or false as first argument to enable or disable actions in settings.json
+setup_actions_enabled_settings_for_restore() {
```

Comment:
we use this to read the data 
```
ACTIONS_ENABLED_IN_BACKUP=$(git config -f $GHE_RESTORE_SNAPSHOT_PATH/settings.json --bool app.actions.enabled | xargs || false)
```
should we use `git config` to set the data in the file?